src/libNode/MicroBlockPreProcessing.cpp

```diff
@@ -1339,12 +1336,10 @@ bool Node::CheckMicroBlockHashes(bytes& errorMsg) {
   // Check Rewards
   if (m_mediator.GetIsVacuousEpoch() &&
       m_mediator.m_ds->m_mode != DirectoryService::IDLE) {
-    // Check COINBASE_REWARD_PER_DS + totalTxnFees
-    uint128_t rewards = 0;
-    if (!SafeMath<uint128_t>::add(m_mediator.m_ds->m_totalTxnFees,
-                                  COINBASE_REWARD_PER_DS, rewards)) {
-      LOG_GENERAL(WARNING, "total_reward addition unsafe!");
-    }
+    // Check COINBASE_REWARD_PER_DS
+
+    uint128_t rewards = COINBASE_REWARD_PER_DS;
```

Comment:
Can just use `COINBASE_REWARD_PER_DS` directly

Reply:
I'm worrying we might have to adjust this number, so easier to maintain if define exceptionally